packages/modules/fulfillment/src/utils/events.ts

```diff
@@ -0,0 +1,293 @@
+import {
+  Fulfillment,
+  FulfillmentSet,
+  GeoZone,
+  ServiceZone,
+  ShippingOption,
+  ShippingOptionRule,
+  ShippingOptionType,
+} from "@models"
+import { Context } from "@medusajs/types"
+import {
+  CommonEvents,
+  eventBuilderFactory,
+  FulfillmentEvents,
+  Modules,
+} from "@medusajs/utils"
+
+export const eventBuilders = {
```

Comment:
@carlos-r-l-rodrigues @sradevski in theory, we could even export that from the utils/fulfillment alongside the event enum wdyt?

Reply:
for now I ll keep it here but I was just thinking out loud

Reply:
Yeah let's keep it for now, it's already well-defined so extracting it will be very easy